server/routes/transfers.js

```diff
@@ -165,6 +166,12 @@ router.post(
       return res.json(transfers);
     } catch (err) {
       console.log('error while creating transfer', err.response.data);
+      if (err.response.data.error_type === 'RATE_LIMIT_EXCEEDED') {
+        return res.status(400).json({
```

Comment:
rate limit is usually 429 instead of 400

Reply:
oh! i did not know this! thanks.